articles/ros2_dds_security.md

```diff
@@ -114,7 +114,7 @@ Let's discuss each of these in turn.
 ### Security files for each domain participant
 
 As stated earlier, the DDS-Security plugins require a set of security files (e.g. keys, governance and permissions files, etc.) per domain participant.
-Domain participants map to a specific instance of a node in ROS 2, so each node requires a set of these files.
+Domain participants map to a process in ROS 2, so each process requires a set of these files.
```

Comment:
Domain participants mat to a [Context](https://github.com/ros2/rclcpp/blob/fd8cfa8fe3e61c9dc2bf2e7918a24c883982b6b6/rclcpp/include/rclcpp/context.hpp#L53).

Reply:
Done: a206b1a